src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationDataSelectionPresenter.cs

```diff
@@ -129,16 +128,5 @@ public void Handle(SimulationReplacedInParameterAnalyzableEvent eventToHandle)
       }
 
       private bool canHandle(SimulationReplacedInParameterAnalyzableEvent eventToHandle) => Equals(eventToHandle.ParameterAnalysable, _parameterIdentification);
-
-      public void Handle(WeightObservedDataChangedEvent eventToHandle)
-      {
-         var existingMapping = _outputMappingPresenter.OutputMappings.Where(x => x.OutputPath == eventToHandle.OutputMapping.OutputPath).FirstOrDefault();
-
-         if (existingMapping == null)
-            return;
-
-         _weightedObservedDataPresenter.Edit(eventToHandle.OutputMapping.WeightedObservedData);
-      }
```

Comment:
this was the part that was bugging me. It's because in practice we are updating one presenter based on the selection in another presenter, and then for this PR we need to react to data that was already replaced.

Reply:
It feels like this presenter knows too much about what the `_outputMappingPresenter` and the `_weightedObservedDataPresenter` are doing.